### ISSUE TITLE
How to get revision number of a working copy (not: Last changed revision)

### ISSUE THREAD

**Issue by patric-r** [martoe/gradle-svntools-plugin#48]
Question here:
Is it possible to get the revision number of a working copy, like `svn info` (first row, revision 900037):

```
svn info
[...]
Revision: 900037
Node Kind: directory
Schedule: normal
Last Changed Author: user
Last Changed Rev: 900000
```

`svntools.info.revisionNumber` seems to return the '`Last Changed Rev`' revision number
which is not what I need.

Thank you.

**Comment by martoe** [comment]
You are right, `svntools.info.revisionNumber` returns the committed revision. To obtain the checked-out revision of the working copy, you may use `svntools.version.maxRevisionNumber` or `svntools.version.maxRevisionNumber` (both containing the same value as long as your working copy doesn't contain mixed revisions)

HTH
Martin
